src/api/app/views/webui/shared/user_or_groups_roles/responsive_ux/_actions.html.haml

```diff
@@ -0,0 +1,9 @@
+- content_for :actions do
+  %li.nav-item
+    = link_to('#', data: { toggle: 'modal', target: '#add-user-role-modal' }, id: 'add-user', class: 'nav-link') do
+      %i.fas.fa-plus-circle
```

Comment:
@dmarcoux I saw that for other font-awesome icons in the action menu, we used ```fa-lg``` class and added a margin to it ```mr-1```.

Reply:
I just checked and you're right. We do use `fa-lg` and `mr-2`.

Reply:
Please have another look.